gettsim/parameters/kinderzuschl.yaml

```
@@ -47,10 +47,10 @@ kinderzuschl_min_eink_alleinerz:
   2008-10-01:
     scalar: 600
     reference: Art. 1 G. v. 24.09.2008 BGBl. I S. 1854.
-kinderzuschl_entzug_eltern:
+entzug_eltern:
```

```suggestion
entzugsrate_eltern:
```
Thinking too much of alcohol otherwise... ;-)